### DISCUSSION TITLE
Put the PDF to your poster as a comment on this Discussion

### DISCUSSION THREAD

**Discussion by jonmatthis** [Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022#77] [General]
please name it something like `firstname_lastname_firstname_lastname_poster.pdf`

**Comment by andreabuitrago1** [comment]
[Ballistic Trajectory of a Human Jump Ellexi & Andrea (1).pdf](https://github.com/Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022/files/9975131/Ballistic.Trajectory.of.a.Human.Jump.Ellexi.Andrea.1.pdf)

**Comment by madelyn-redick** [comment]
data luverz <3
[data luverz - madelyn redick, sahana dhar, shreya srinivasan.pdf](https://github.com/Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022/files/9975133/data.luverz.-.madelyn.redick.sahana.dhar.shreya.srinivasan.pdf)

**Reply by madelyn-redick** [reply]
https://northeastern-my.sharepoint.com/:p:/g/personal/redick_m_northeastern_edu/Ecq-_ozwrA1JoyBZDJ4ipoMB80vaAkdnkB5NOYJGC0TG_g

**Comment by Julianlee24** [comment]
[Julian_Lee_Sahana_Dhar.pdf](https://github.com/Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022/files/9975137/Julian_Lee_Sahana_Dhar.pdf)

**Comment by saanvi-vutukur** [comment]
[neural control standing posture.pdf](https://github.com/Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022/files/9975238/neural.control.standing.posture.pdf)

neural control of standing posture

**Reply by glor-n** [reply]
[Posters Slides.pdf](https://github.com/Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022/files/9975972/Posters.Slides.pdf)

**Comment by Robinsbank** [comment]
[robin_bank_poster.pdf](https://github.com/Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022/files/9975248/robin_bank_poster.pdf)
